apps/client/src/types/index.ts

```diff
@@ -0,0 +1,18 @@
+export interface BaseEntity {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+export interface User extends BaseEntity {
+  first: string;
+  last: string;
+  roleId: string;
+  photo: string;
+}
+
+export interface Role extends BaseEntity {
+  name: string;
+  isDefault: boolean;
+  description: string;
+}
```

Comment:
In a better world these types would be shared across the server and client apps...